src/fragments/lib/push-notifications/ios/getting_started/20_set_entitlements.mdx

```diff
@@ -0,0 +1,65 @@
+## Set Entitlements
+
+Using Amazon Pinpoint with APNs requires the following capabilities
+- Push Notifications
+- Background Processing -> Remote Notifications (**iOS only**)
+- App Sandbox -> Outgoing Connections (**macOS only**)
+
+To add these capabilities:
+
+<BlockSwitcher>
+<Block name="iOS">
+
+1. In project settings, select your main application target, select the **Signing and Capabilities** tab, and click **+ Capability** in the upper left corner of the editor pane to bring up the Capabilities dialog.
```

Comment:
Maybe this is inherrent to a Swift developer - but can we specify you are doing this in Xcode?

Reply:
added